cmake/compiler/gcc/target.cmake

```diff
@@ -1,8 +1,11 @@
-set_ifndef(CC gcc)
 set_ifndef(C++ g++)
```

Comment:
for consistency, I think it would be better to also removing if-not-def'd C++ too.

Reply:
This would violate the design.

generic.cmake contains just enough configuration to run a preprocessor on DT files. And target.cmake contains the rest.